src/client/pps/pps.proto

```diff
@@ -308,6 +312,7 @@ service API {
   rpc InspectJob(InspectJobRequest) returns (JobInfo) {}
   rpc ListJob(ListJobRequest) returns (JobInfos) {}
   rpc DeleteJob(DeleteJobRequest) returns (google.protobuf.Empty) {}
+  rpc StopJob(StopJobRequest) returns (google.protobuf.Empty) {}
```

Comment:
@jdoliner I think distinguishing between `STOPPED` and `FAILURE` makes sense.  `FAILURE` is when things actually went wrong.  `STOPPED` is when you manually make the job stop, even though it might succeed eventually.